apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/project.json

```diff
@@ -24,6 +24,6 @@
 
   "frameworks": {
     "net45": {},
-    "netstandard1.5": {}
+    "netcoreapp1.5": {}
```

Comment:
Surely it really *should* be netstandard1.5, no? It's not an app, it's a library...